packages/shared/styles/components/molecules/SfAddToCart.scss

```diff
@@ -1,34 +1,27 @@
 @import '../../variables';
-:root {
-  --add-to-cart__select-quantity-min-width: 4.625rem;
-  --add-to-cart__select-quantity-max-width: 5.625rem;
-}
+
 .sf-add-to-cart {
   display: flex;
   &__button {
     flex: 1;
+    @include for-desktop{
+      order: 1;
+    }  
   }
   &__select-quantity {
+    --input-height: auto;
     display: flex;
-    margin: 0 0 0 .5rem;
-    height: auto;
-    min-width: var(--add-to-cart__select-quantity-min-width);
-    max-width: var(--add-to-cart__select-quantity-max-width);
-    input {
-      border: 1px solid var(--c-light);
-    }
-  }
-}
-@media screen and (min-width: $desktop-min) {
-  .sf-add-to-cart {
-    &__button {
-      order: 1;
+    min-width: var(--add-to-cart-select-quantity-min-width, 4.625rem);
+    max-width: var(--add-to-cart-select-quantity-max-width, 5.625rem);
+    margin: var(--add-to-cart-select-quantity-margin, 0 0 0 .5rem);
+    @include for-desktop {
+      --add-to-cart-select-quantity-margin: 0;
     }
-    &__select-quantity {
-      margin-left: 0;
-      input {
-        font-size: var(font-size-extra-big);
-        background-color: var(--c-light);
+    input {
+      border-color: var(--add-to-cart-select-quantity-input-border-color, var(--c-light));
+      @include for-desktop {
+        font-size: var(--add-to-cart-select-quantity-input-f-size, var(--font-size-extra-big));
```

Comment:
@martaradziszewska use always font() when you want need even one font property example rules 19 in our docs
```
--input-font-size: var(--font-size-extra-big)
```
and for border
```
input{
--input-border: 1px solid var(--c-light)
}
```